utils/database.js

```diff
@@ -10,17 +10,11 @@ class Database {
       if (db !== undefined) {
         return resolve(true)
       }
-      new Promise.resolve(SQLite.deleteDatabase(
-        {
-          name: database_name,
-          location: 1,
-        })).finally(() =>
-          setTimeout(function () {
-            db = SQLite.openDatabase({ name: database_name, createFromLocation: 1 });
+      db = SQLite.openDatabase({ name: database_name, createFromLocation: 1 },()=>{console.log("Database open Successfully")},
+      (error)=>{console.log("Error",error)});
```

Comment:
Veerji, please break this line into shorter lines so that it can be readable.

Reply:
Fixed Veer ji